mlir/lib/Dialect/Tosa/Utils/ConversionUtils.cpp

```diff
@@ -213,3 +213,29 @@ mlir::tosa::convertFromIntAttr(const DenseElementsAttr &attr, const int rank) {
   }
   return {};
 }
+
+Attribute mlir::tosa::getConstantAttribute(Operation *op) {
+  if (op == nullptr) {
+    return Attribute();
+  }
+
+  if (isa<ConstOp>(op)) {
+    return cast<ConstOp>(op).getValues();
+  }
+
+  // Very strong assumption, but most constants from all dialects store the
+  // value of the constant in within a value key. Here we check for that, if it
+  // does not have that attribute then we bail.
+  assert(op->hasTrait<OpTrait::ConstantLike>() &&
+         "the operation must be a constant.");
+
+  // TOSA names constants in the operation as "value" while linalg names them
+  // with "values". Here we search for both and find the first.
+  const SmallVector<const char *> possibleAttributes = {"value", "values"};
+  for (const char *name : possibleAttributes) {
```

Comment:
```suggestion
  for (StringRef name : possibleAttributes) {
```
...as it [will be used](https://mlir.llvm.org/doxygen/classmlir_1_1Operation.html#a505ceebadfa1e23e9c3bfbec05e20f66) to pass the argument anyway.

Reply:
done